src/cmdfuncs.lua

```diff
@@ -565,6 +565,12 @@ function Restore(collection)
 
 
    local masterTbl = masterTbl()
+   local force
+   if (masterTbl.force) then
```

Comment:
what's the point of this?
```lua
local force = masterTbl.force
```
does the same? And if you only need to pass it as an argument, do you really need a local variable?